feature_engine/imputation/na_transformer.py

```diff
@@ -0,0 +1,192 @@
+# Authors: Pradumna Suryawanshi <pradumnasuryawanshi@gmail.com>
+# License: BSD 3 clause
+
+from typing import Optional, List, Union
+
+import pandas as pd
+
+from feature_engine.dataframe_checks import _is_dataframe
+from feature_engine.imputation.base_imputer import BaseImputer
+from feature_engine.parameter_checks import _define_numerical_dict
```

Comment:
you are not using this import I think. This will make style_checks fail. Please remove.